src/Traits/DetectsChanges.php

```diff
@@ -24,13 +24,21 @@ protected static function bootDetectsChanges()
 
     public function attributesToBeLogged(): array
     {
-        if (! isset(static::$logAttributes)) {
+        if (!isset(static::$logAttributes)) {
             return [];
         }
 
-        return static::$logAttributes;
+        return collect(static::$logAttributes)->map(
+            function ($value) {
+                if (strpos($value, '.') != 0) {
```

Comment:
This checks if there are dots in the string for example `example.deeper`.
`starts_with` would not catch these right?